packages/rest/src/__tests__/unit/parser.unit.ts

```diff
@@ -346,6 +346,19 @@ describe('operationArgsParser', () => {
       );
     });
 
+    it('parses url-encoded complex json object', async () => {
+      const req = givenRequest({
```

Comment:
nitpick: do we have to use an encoded url with `%7B`, %3A, etc..?

Reply:
@jannyHou using the encoded url, is an explicit way of testing if the same query request will work from api explorer. otherwise supertest implicitly converts json query objects to url encoded request.